network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -870,7 +869,7 @@ def product_view(self, request, slug):
         # Find product by it's slug and redirect to the product page
         # If no product is found, redirect to the BuyersGuide page
         product = get_object_or_404(ProductPage, slug=slug)
-        return redirect(product.url)
+        return redirect(product.url, permanent=True)
```

Comment:
We can't know if this is permanent - someone might be trying to access a product that's already linked in some blog post that we're still in the process of adding and it won't be live until a week from now, so let's not make this a permanent redirect.